scripts/dockerfiles/Dockerfile.static

```diff
@@ -1,5 +1,6 @@
 ARG GO_VERSION
 FROM golang:$GO_VERSION as builder
+RUN useradd -u 10001 agentuser
```

Comment:
any reason, not adding group here.

Reply:
group is not really needed but i will add it